### PR TITLE
Update features to add new affect

### PR DESCRIPTION
**Pull request by qduanmu**
OSIDB-1859

### CODE REVIEW

File: features/flaw_detail.feature [hunk 1]
```diff
@@ -110,3 +110,8 @@ Feature: Flaw detail testing
       Given I go to an embargoed flaw detail page
       When I update the embargoed flaw with a future public date
       Then The embargoed flaw is updated
+
+    Scenario: Add new affect to flaw
+      Given I go to a public flaw detail page
+      When I add a new affect with valid data
+      Then The affect is added
```
Comment (RichardXuan) [inline features/flaw_detail.feature:117]: Meet "Flaw validation failed before submission
20 seconds ago
affects/3/cvss_scores/0/affect: Required
 affects/3/cvss_scores/0/comment: Required" error info. Is this a bug? Using flaw "CVE-2023-2023"

File: features/steps/flaw_detail.py [hunk 2]
```diff
@@ -445,6 +445,25 @@ def step_impl(context):
...
+
+
+@when("I add a new affect with valid data")
+def step_impl(context):
+    go_to_specific_flaw_detail_page(context.browser)
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.click_button_with_js('addNewAffectBtn')
+    context.ps_component = flaw_detail_page.set_new_affect_inputs()
+    flaw_detail_page.click_btn('saveBtn')
+    flaw_detail_page.wait_msg('affectCreatedMsg')
```
Comment (huiwangredhat) [inline features/steps/flaw_detail.py:461]: Does the adding affect work well on the current stage?
I tried to run the feature for CVE-2023-2023, but it gave the following error. I went to the selenium page, I saw the saving has been running...
 _seleniumpagefactory.Pagefactory.ElementNotFoundException: An exception of type TimeoutException occurred. With Element -: affectCreatedMsg - locator: (xpath, //div[text()='Affect Created.'])_

File: features/steps/flaw_detail.py [hunk 3]
```diff
@@ -445,6 +445,25 @@ def step_impl(context):
...
+
+
+@then("The affect is added")
+def step_impl(context):
+    go_to_specific_flaw_detail_page(context.browser)
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.check_value_exist(context.ps_component)
```
Comment (huiwangredhat) [inline features/steps/flaw_detail.py:468]: Here is only checked the component that is generated by random. That's ok. But a miner thing, the affect can't save the cvssv3 even though it was set in the above stage.

**Comment by huiwangredhat** [comment]
Others LGTM.

File: features/steps/flaw_detail.py (hunk 2 above)
Comment (huiwangredhat) [reply features/steps/flaw_detail.py:461]: After discussing off line, this is caused by the test data.

**Review by huiwangredhat — APPROVED** [review]

**Pull request merged**